README.md

```diff
@@ -93,6 +93,22 @@ This site exposes a public JSON API powered by [Next.js API routes](https://next
 - [`/api/r/:emoji`](https://scrapbook.hackclub.com/api/r/hardware) – Get all posts tagged with a specific emoji
 - [`/:username.png`](https://scrapbook.hackclub.com/sampoder.png) – Get a user's avatar as an image URL
 
+## Running Locally
+
+1. Clone this repository
+   - `git clone git@github.com:hackclub/scrapbook.git; cd scrapbook`
+1. Install dependencies
+   - `yarn`
+1. Link Vercel to your account for deployment
+   - `yarn vercel`
+   - _Note_: The project name in Vercel is `summer-streaks`
+1. Pull environment variables from Vercel
+   - `yarn vercel env pull`
```

Comment:
```suggestion
   - `yarn vercel pull`
```

<img width="655" alt="Screen Shot 2022-03-02 at 10 21 44" src="https://user-images.githubusercontent.com/72365100/156424890-890b9ee4-587a-4b1e-a1ef-b231ca991015.png">

Reply:
@exu3 Nice, I didn't notice that 🔍  Fixed in [4477ed4](https://github.com/hackclub/scrapbook/pull/280/commits/4477ed44ccfdfad78ac6b2c0fb52730d67d7110a)